ingestion/src/metadata/orm_profiler/profiler/core.py

```diff
@@ -213,6 +216,10 @@ def custom_metrics(self) -> List[Type[CustomMetric]]:
     def query_metrics(self) -> List[Type[QueryMetric]]:
         return self._filter_metrics(QueryMetric)
 
+    @property
+    def system_metrics(self) -> List[Type[SystemtMetric]]:
```

Comment:
Aren't system metrics just a flavour of `QueryMetric`s (e.g., the histogram)?

Reply:
kind of, but Query metrics are querying specific columns for that table. We don't query the table but the information schema table.

Reply:
oooh got it. Makes sense, We could eventually use this SystemMetric to query internal statistics etc. Thank you for explaining

Reply:
also I just noticed there's a typo `List[Type[SystemtMetric]]:` System"t" metric